dashboard/v1.0/src/components/layouts/GlobalLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Header from './Header';
+import Sidebar from './Sidebar';
+
+export default props => {
```

Comment:
props keyword here can kind of mislead newbies to react. We can change it to the component name maybe. @aquibbaig

Reply:
Agreed. How about `root`? as a root component in the react-dom.

Reply:
Thanks @ganeshpatro321!